html/inc/boinc_db.inc

```diff
@@ -823,6 +835,50 @@ class BoincToken {
         return $db->affected_rows();
     }
 
+    static function delete_for_user($user_id) {
+        $db = BoincDb::get();
+        $now = time();
+        $db->delete_aux('token', "userid=$user_id");
+        return $db->affected_rows();
+    }
+
+}
+
+class BoincUserDeleted {
+    
+    static function insert_user($user) {
+        $now = time();
+        $cpid = md5($user->cross_project_id.$user->email_addr);
+        $clause = "(userid, public_cross_project_id, create_time) values ($user->id, '$cpid', $now)";
+        $db = BoincDb::get();
+        return $db->insert('user_deleted', $clause);
+    }
+    
+    static function delete_expired() {
+        $db = BoincDb::get();
+        $expire_time = time() - 2*30*86400; //60 days ago
```

Comment:
No sure why 60 needs to be expressed as 2*30 🤔 (same for host below)

Reply:
changed